.eslintrc

```diff
@@ -1,14 +1,21 @@
 {
-  "extends": "vtex-react/gatsby",
+  "extends": [
+    "vtex-react/gatsby",
```

Comment:
Can we also remove this?

Reply:
not really. We need to have our own linting rules :(